src/Symfony/Component/AssetMapper/AssetMapperDevServerSubscriber.php

```diff
@@ -134,17 +139,28 @@ public function onKernelRequest(RequestEvent $event): void
             ->setPublic()
             ->setMaxAge(604800)
             ->setImmutable()
-            ->setEtag($asset->digest)
-        ;
+            ->setEtag($asset->digest);
+
+        $response->headers->set('X-AssetMapper-Server', true);
```

Comment:
just bikeshedding :)
```suggestion
        $response->headers->set('X-DevMappedAsset', true);
```

Reply:
✅  🚲